newIDE/app/src/ProjectsStorage/index.js

```diff
@@ -10,6 +10,7 @@ import { type AppArguments } from '../Utils/Window';
 export type FileMetadata = {|
   /** The file id, path or local path according to the provider. */
   fileIdentifier: string,
+  projectName?: string,
```

Comment:
should this be nullable?

Reply:
I have not encountered a case in which it is useful.